Installing-and-Configuring/Prerequisites.md

```diff
@@ -74,66 +74,49 @@ USB Security Dongles (aka security token aka smartcard)
 
 *NOTE* - If you prefer not to use USB security dongles or want simplified security procedures, see the [Purism Boot Modes](/PurismBootModes) documentation for information about Basic and Restricted boot modes that provide different security/usability trade-offs.
 
-Legacy vs Maximized boards
+Board Architecture Overview
 ===
-Some history first on the historical x230-flash and x230 boards that initially created the Heads project.
 
-Heads was initially developped on the x230 board (first xx30 board supported).
+**Note**: All current Heads boards use a modern architecture where the Intel Management Engine (ME) is neutered and the Intel Flash Descriptor (IFD) is unlocked. The historical distinction between "Legacy" and "Maximized" boards is no longer relevant as of 2024, since all supported boards now use the approach that was previously called "maximized."
```

Comment:
Untrue. Not all boards provide neutered ME which is only possible (up to now) up to Ivy bridge (Intel 3rd gen). Maybe you should digest and make sense of https://github.com/linuxboot/heads/issues/692 which details blob level for each chipset revision and ME requirements,

It is true to say that all Heads boards produced rom artifact is whole and valid, and also true to say that packed ME blobs at build is deactivated. Not all rom produced are neutered since its impossible from skylake on. Getting into the details of HAP bits and all is irrelevant here, unless you can synthetize nicely https://github.com/linuxboot/heads/issues/692 content and make sense of it to be useful here, which would be nice but not necessary. Just don't say false things please.